### PR TITLE
12 school fees

### PR DESCRIPTION
**Pull request by AnuKPatel**
Hello,

Migration of school_fees module is done.
Please review and merge the code.

### CODE REVIEW

File: school_fees/views/school_fees_view.xml [hunk 1]
```diff
@@ -254,8 +254,8 @@
         <field name="arch" type="xml">
             <search string="Payslip">
                 <group col="10" colspan="4">
-                    <filter icon="terp-document-new" string="Draft" domain="[('state','=','draft')]" help="Draft Slip"/>
-                    <filter icon="terp-camera_test" string="Confirm" domain="[('state','=','confirm')]" help="Confirm Slip"/>
+                    <filter icon="terp-document-new" name="draft" string="Draft" domain="[('state','=','draft')]" help="Draft Slip"/>
```
Comment (dhirendrasinh-Serpent) [inline school_fees/views/school_fees_view.xml]: No need icon attribute in filter V12

File: school_fees/views/school_fees_view.xml [hunk 2]
```diff
@@ -254,8 +254,8 @@
...
+                    <filter icon="terp-camera_test" name="confirm" string="Confirm" domain="[('state','=','confirm')]" help="Confirm Slip"/>
```
Comment (dhirendrasinh-Serpent) [inline school_fees/views/school_fees_view.xml]: No need icon attribute in filter V12

File: school_fees/views/school_fees_view.xml [hunk 3]
```diff
@@ -352,8 +352,8 @@
         <field name="arch" type="xml">
             <search string="Fees Register">
                 <group col="10" colspan="4">
-                    <filter icon="terp-document-new" string="Draft" domain="[('state','=','draft')]" help="Draft Slip"/>
-                    <filter icon="terp-camera_test" string="Confirm" domain="[('state','=','confirm')]" help="Confirm Slip"/>
+                    <filter icon="terp-document-new" name="draft" string="Draft" domain="[('state','=','draft')]" help="Draft Slip"/>
```
Comment (dhirendrasinh-Serpent) [inline school_fees/views/school_fees_view.xml]: No need icon attribute in filter V12

File: school_fees/views/school_fees_view.xml [hunk 4]
```diff
@@ -352,8 +352,8 @@
...
+                    <filter icon="terp-camera_test" name="confirm" string="Confirm" domain="[('state','=','confirm')]" help="Confirm Slip"/>
```
Comment (dhirendrasinh-Serpent) [inline school_fees/views/school_fees_view.xml]: No need icon attribute in filter V12

File: school_fees/views/school_fees_view.xml [hunk 5]
```diff
@@ -416,27 +416,15 @@
             <data>
                 <xpath expr="//div[@name='buttons']" position="inside">
                     <button name="%(action_stu_fees_form)d" type="action" class="oe_inline oe_stat_button" attrs="{'invisible':[('state', 'in', ['draft','terminate','alumni','cancel'])]}" groups="school.group_school_administration,school.group_school_student">
-                        <img src="/school_fees/static/description/icon.png" style="height:25px;width:50px;float:left;"/>
+                        <img src="/school_fees/static/description/icon.png" alt="logo"
+                            style="height:25px;width:50px;float:left;"/>
                         <lable style="">Fees Receipt</lable>
```
Comment (dhirendrasinh-Serpent) [inline school_fees/views/school_fees_view.xml:421]: No need style attribute

**Review by dhirendrasinh-Serpent — COMMENTED** [review]
Improve code

File: school_fees/views/school_fees_view.xml (hunk 5 above)
Comment (AnuKPatel) [reply school_fees/views/school_fees_view.xml:421]: It is needed.

**Comment by AnuKPatel** [comment]
Removed icon from all the filter tags.

**Pull request merged**